cmd/world/cardinal/dev.go

```diff
@@ -83,6 +83,9 @@ var devCmd = &cobra.Command{
 			cmdErr <- err
 		}()
 
+		fmt.Println("Cardinal Started..")
```

Comment:
Why not use the logger helpers for this and the log lines below?

Reply:
logger helpers is only printed when using --degub / debug mode
in this case, i want to print in every situation.

Reply:
I personally think the specific logs that @zulkhair added here are OK to be printed regardless of debug level given this tool is meant for the first touch with world engine. Especially the one that warns user's about the `--debug` flag. 

Doesn't look like there is a way to override the log level set in the logger in a nice way on first look

Reply:
👍